packages/circuits/config/index.ts

```diff
@@ -7,23 +9,13 @@ export const proveReputationCircuitPath =
 export const proveUserSignUpCircuitPath =
     '../zksnarkBuild/proveUserSignUp_main.circom'
 
-export const startTransitionCircuitPath =
-    '../zksnarkBuild/startTransition_main.circom'
-
-export const processAttestationsCircuitPath =
-    '../zksnarkBuild/processAttestations_main.circom'
-
-export const userStateTransitionCircuitPath =
-    '../zksnarkBuild/userStateTransition_main.circom'
```

Comment:
These can also be removed?

Reply:
cfd4ed12797ab5c78bf59d8407a6bd33a8e9264e